### PR TITLE
Upgrade lodash to 4.17.21

### PR DESCRIPTION
**Pull request by yukipastelcat**
Ticket: https://oat-sa.atlassian.net/browse/ADF-1391

## What's Changed

- [lodash](https://www.npmjs.com/package/lodash) dependency been upgraded to 4.17.21 w/o locking the version
- Upgrade has been done with `--lockfile-version 1` flag to avoid changing lockfile structure
- `_.contains` has been replaced with `_.includes` as it was changed in [3.10.1](https://lodash.com/docs/3.10.1#includes) and alias was removed in [4.17.21](https://lodash.com/docs/4.17.15#includes)

## How to test
- Run `npm run build` in order to check that build isn't broken
- Run `npm test` to check qunit-testrunner tests

### CODE REVIEW

**Review by ViktoriaKh-oat — APPROVED** [review]
Looks good! Thanks!
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Review by btamas — CHANGES_REQUESTED** [review]
This is a breaking change.

**Comment by yukipastelcat** [comment]
@btamas why? This is not breaking the API of the package, it remains unchanged, behaviour of the methods is also unchanged

**Review by btamas — APPROVED** [review]

**Comment by btamas** [comment]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Commits messages are meaningful
- [ ] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
         "handlebars": "1.3.0",
         "interactjs": "1.3.4",
         "jquery": ">= 1.9.1 < 3.0.0",
-        "lodash": "2.4.1",
+        "lodash": "^4.17.21",
```
Comment (jsconan) [inline package.json:51]: **issue:** Freaking! Did you check if other libs were aligned with this before merging? :o

Comment (btamas) [reply package.json:51]: I've checked and there are only ~10 `_.` calls, like defer, forIn or extend. All of them exists in the new version.

**Pull request merged**